tests/unit/config/test_configfiles.py

```diff
@@ -134,6 +134,144 @@ def test_yaml_oserror(fake_save_manager, config_tmpdir):
     assert error.traceback is None
 
 
+class TestConfigPyModules:
+
+    """Test for ConfigPy Modules."""
```

Comment:
You can remove this docstring - I made them optional for tests, because often they look like this, and say the same thing class names say already.